ChangeLog

```diff
@@ -25,6 +25,10 @@ Release date: TBA
 
   Closes PyCQA/pylint#5771
 
+* Improve location of keyword-only arguments in ``astroid.scoped_nodes.Lambda.argnames``.
+
+  Refs PyCQA/pylint#5771
+
```

Comment:
That is not necessary. We haven't release a new version since, so the entry for the initial change is enough.